### ISSUE TITLE
Не нужен ли комментарий при принятии решения по арбитражу?

### ISSUE THREAD

**Issue by Fenslau** [tebaly/freedomsex#126]
http://hdd.tomsk.ru/desk/pqqoqnjz#
Читаем комментарий от сбросившего в арбитраж. Но допустим, не согласны с ним. Там написано "Привет,..." , а значит это знакомства с целью вирта - первое предложение, так сказать. Ну так вот, допустим если есть желание этот коммент запостить при принятии решения, может быть сделать для этого возможность?

**Comment by mj4444ru** [comment]
ничего не понял. хотите вместо арбитража напрямую перемещать?

**Comment by Fenslau** [comment]
Нет, предлагаю решение по арбитражу сопровождать комментарием. Иногда полезно бывает. Опционально.

**Comment by tebaly** [comment]
* Можно на JS своять форму "Добавить комментарий"
* Добавить ее на все этапы выше голосования: Решение + Арбитраж + Вердикт

**Comment by mj4444ru** [comment]
в решение ничего не надо добавлять только...
